package.json

```diff
@@ -93,6 +93,9 @@
     "@emotion/styled": "^10.0.10",
     "date-fns": "^1.30.1",
     "keycode": "^2.2.0",
+    "lodash.map": "^4.6.0",
```

Comment:
Based on the usage we can probably use the native `map/reduce` instead
to save some bits on the package size